OzStripsGUI/OzStripsGUI.csproj

```diff
@@ -12,6 +12,6 @@
 		<PackageReference Include="SocketIOClient" Version="3.1.1" />
 	</ItemGroup>
 	<ItemGroup>
-		<Reference Include="$(MSBuildProgramFiles32)\vatSys\bin\vatSys.exe "/>
+		<Reference Include="$(VatSysDir)\bin\vatSys.exe " />
```

Comment:
This is where the build magic comes in, by default it'll pick up the program files 86 directory, on the build server it'll pick up where the vatsys download has happened 

Nice thing also in theory this should handle the case where vatsys updates, and things may have changed.